osbrain/logging.py

```diff
@@ -30,14 +30,14 @@ def on_init(self):
         self.log_history_debug = []
         self.log_history = []
         handlers = {
-            'INFO': self.log_handler,
-            'WARNING': self.log_handler,
-            'ERROR': self.log_handler,
-            'DEBUG': self.log_handler
+            'INFO': self._log_handler,
+            'WARNING': self._log_handler,
+            'ERROR': self._log_handler,
+            'DEBUG': self._log_handler
         }
         self.bind('SUB', 'sub', handlers)
 
-    def log_handler(self, message, topic):
```

Comment:
I would not set `log_handler` as private. It is more like a dummy method that the user would probably want to override anyway (unless until we implement better logging...).